roles/container-engine/docker/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-docker_version: '24.0'
+docker_version: '26.1'
```

Comment:
HI @tico88612 

Is it better to split it into two PRs, 
One supports Ubuntu 24.04, and another upgrades the Docker version.

Reply:
PR #11198

I will rebase after #11198 is merged.

Reply:
Rebased.